src/base/constants.py

```diff
@@ -44,3 +44,16 @@ def _IsDevAppServer():
 DEBUG = IS_DEV_APPSERVER
 
 TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+
+AUTHORIZATION_METHOD = 'basic_auth'
```

Comment:
I understood it in the way of executing program, which means once you set them, they are not changed while the code is running.